RecoTracker/TkTrackingRegions/plugins/AreaSeededTrackingRegionsBuilder.h

```diff
@@ -89,25 +91,27 @@ class AreaSeededTrackingRegionsBuilder {
 
     const AreaSeededTrackingRegionsBuilder *m_conf = nullptr;
     const MeasurementTrackerEvent *m_measurementTracker = nullptr;
+    TrackingSeedCandidates::Objects candidates;
   };
 
   AreaSeededTrackingRegionsBuilder(const edm::ParameterSet& regPSet, edm::ConsumesCollector&& iC): AreaSeededTrackingRegionsBuilder(regPSet, iC) {}
   AreaSeededTrackingRegionsBuilder(const edm::ParameterSet& regPSet, edm::ConsumesCollector& iC);
   ~AreaSeededTrackingRegionsBuilder() = default;
-
+  
   static void fillDescriptions(edm::ParameterSetDescription& desc);
 
   Builder beginEvent(const edm::Event& e) const;
 
-private:
-  std::vector<Area> m_areas;
-
+  private:
+   std::vector<Area> m_areas;
+   TrackingSeedCandidates candidates_;
   float m_extraPhi;
   float m_extraEta;
   float m_ptMin;
   float m_originRadius;
   bool m_precise;
   edm::EDGetTokenT<MeasurementTrackerEvent> token_measurementTracker;
+  edm::EDGetTokenT<TrackingSeedCandidates> token_candidates;
```

Comment:
I believe this is not needed (not used, and the token for candidates is already a member of `TrackingSeedCandidates`).